.github/workflows/ci.yml

```diff
@@ -12,13 +12,13 @@ jobs:
     strategy:
       matrix:
         ruby:
-          - 2.7.5
           - 3.0.3
           - 3.1.1
+          - 3.2.2
```

Comment:
**suggestion:** Can we complete the list of 3.x version here?

```suggestion
          - 3.0
          - 3.1
          - 3.2
          - 3.2
```

Reply:
done, thanks for the suggestion ben

Reply:
Only if we can @chigginsboost. If we can't go up to `3.3`, we can leave it up to `3.2`

Reply:
okay yeah unfortunately it is erroring at 3.3, I've reverted this for now. maybe we can look into it later